### PR TITLE
NTBS-3052 Unmatched Specimens Report

### PR DESCRIPTION
**Pull request by jordan-carlton**
Remove primary key from Linked Notifications table following QA feedback

### CODE REVIEW

File: source/dbo/Tables/Report Unmatched Specimens/UnmatchedSpecimensLinkedNotifications.sql [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
-	[NotificationStatus] NVARCHAR(30),
-	[NhsNumber] NVARCHAR(10),
-	[BirthDate] DATETIME2(7),
-	[FamilyName] NVARCHAR(35),
-	[GivenName] NVARCHAR(35),
-	[Sex] NVARCHAR (200),
-	[Address] NVARCHAR (150),
-	[Postcode] NVARCHAR(10),
-	[RegionCode] NVARCHAR(50)
+	[Denotified] NVARCHAR(30),
```
Comment (LeoCie) [inline source/dbo/Tables/Report Unmatched Specimens/UnmatchedSpecimensLinkedNotifications.sql:8]: Is there supposed to be a change to the proc that adds to this table as part of this commit? I don't think denotified is a good name if it's still going to contain the status.

File: source/dbo/Tables/Report Unmatched Specimens/UnmatchedSpecimensLinkedNotifications.sql [hunk 2]
```diff
@@ -1,17 +1,16 @@
 ﻿CREATE TABLE [dbo].[UnmatchedSpecimensLinkedNotifications]
 (
-	[ReferenceLaboratoryNumber] NVARCHAR(50) NOT NULL PRIMARY KEY,
+	[ReferenceLaboratoryNumber] NVARCHAR(50) NOT NULL,
```
Comment (JoshCadney) [inline source/dbo/Tables/Report Unmatched Specimens/UnmatchedSpecimensLinkedNotifications.sql:3]: Should we be replacing this with a primary key on the `ReferenceLaboratoryNumber` and the `NotificationID`?

Perhaps we shouldn't, as it's at least conceptually possible that the same notification could be related to a lab result in two different ways

**Review by JoshCadney — CHANGES_REQUESTED** [review]
No mark-ups from me, but it looks like Leon's point needs addressing

File: source/dbo/Tables/Report Unmatched Specimens/UnmatchedSpecimensLinkedNotifications.sql (hunk 1 above)
Comment (jordan-carlton) [reply source/dbo/Tables/Report Unmatched Specimens/UnmatchedSpecimensLinkedNotifications.sql:8]: I didn't mean to include these changes (they were from a change I made to not join to NTBS directly, but then discarded)

**Review by LeoCie — APPROVED** [review]
Looks good now!

**Pull request merged**